homeassistant/components/epson/__init__.py

```diff
@@ -1 +1,70 @@
-"""The epson component."""
+"""The epson integration."""
+import asyncio
+import logging
+
+from epson_projector import Projector
+from epson_projector.const import POWER
+
+from homeassistant.components.media_player import DOMAIN as MEDIA_PLAYER_PLATFORM
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_PORT, STATE_UNAVAILABLE
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN
+from .exceptions import CannotConnect
+
+PLATFORMS = [MEDIA_PLAYER_PLATFORM]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def validate_projector(hass: HomeAssistant, host, port):
+    """Validate the given host and port allows us to connect."""
+    epson_proj = Projector(
+        host=host,
+        websession=async_get_clientsession(hass, verify_ssl=False),
+        port=port,
+    )
+    _power = await epson_proj.get_property(POWER)
+    if not _power or _power == STATE_UNAVAILABLE:
```

Comment:
We should not use a home assistant state constant and compare that with the device library api. Coupling the two different APIs like that is fragile and may break if any of the APIs change. Please define a local constant to compare with the library API.